portfolio.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <a href="./index.html">WebStudio</a>
+      <nav>
+        <ul>
+          <li>
+            <a href="./index.html">студія</a>
+          </li>
+          <li>
+            <a href="#">портфоліо</a>
+          </li>
+          <li>
+            <a href="#">контакти</a>
+          </li>
+        </ul>
+      </nav>
+
+      <div>
+        <a href="mailto:info@devstudio.com">info@devstudio.com</a>
+        <a href="tel:+38 096 111 11 11">+38 096 111 11 11</a>
+      </div>
+    </header>
+
+    <main>
+      <section>
+        <ul>
+          <li>
+            <button type="button">Усі</button>
+          </li>
+          <li>
+            <button type="button">Веб-сайти</button>
+          </li>
+          <li>
+            <button type="button">Додатки</button>
+          </li>
+          <li>
+            <button type="button">Дизайн</button>
+          </li>
+          <li>
+            <button type="button">Маркетинг</button>
+          </li>
+        </ul>
+      </section>
```

Comment:
as this block is logically connected to the project block, you may omit using separate `section` for it. Just add this list into `section` below, having only one `section` on the `Portfolio` page